[2017-11-23  13:58:57] :  - 
 _      ____       ______Python-AGENT  
| | /| / / /  ___ /_  __/__ ____
| |/ |/ / _ \/ _ `// / / _ `/ _ \
|__/|__/_//_/\_,_//_/  \_,_/ .__/
                          /_/
Just Tap, Always Monitoring
WhaTap Python Agent version 0.1.48, 20171117

WHATAP_HOME: /Users/kimjihye/Downloads/django-react-blog-master/whatap
Config: /Users/kimjihye/Downloads/django-react-blog-master/whatap/whatap.conf
Logs: /Users/kimjihye/Downloads/django-react-blog-master/whatap/logs


[2017-11-23  13:58:57] : WA000 - WHATAP AGENT START!
[2017-11-23  13:59:18] :  - 
 _      ____       ______Python-AGENT  
| | /| / / /  ___ /_  __/__ ____
| |/ |/ / _ \/ _ `// / / _ `/ _ \
|__/|__/_//_/\_,_//_/  \_,_/ .__/
                          /_/
Just Tap, Always Monitoring
WhaTap Python Agent version 0.1.48, 20171117

WHATAP_HOME: /Users/kimjihye/Downloads/django-react-blog-master/whatap
Config: /Users/kimjihye/Downloads/django-react-blog-master/whatap/whatap.conf
Logs: /Users/kimjihye/Downloads/django-react-blog-master/whatap/logs


[2017-11-23  13:59:18] : WA000 - WHATAP AGENT START!
[2017-11-23  04:59:19] :  - type object 'BaseHandler' has no attribute 'apply_response_fixes'
Traceback (most recent call last):
  File "/Users/kimjihye/Downloads/env/lib/python3.4/site-packages/whatap/trace/trace_import.py", line 29, in load_module
    IMPORT_HOOKS[fullname]['def'])(module)
  File "/Users/kimjihye/Downloads/env/lib/python3.4/site-packages/whatap/trace/mod/application_django.py", line 59, in instrument_handlers_base
    module.BaseHandler.apply_response_fixes)
AttributeError: type object 'BaseHandler' has no attribute 'apply_response_fixes'
[2017-11-23  04:59:19] :  - 
 _      ____       ______Python-AGENT  
| | /| / / /  ___ /_  __/__ ____
| |/ |/ / _ \/ _ `// / / _ `/ _ \
|__/|__/_//_/\_,_//_/  \_,_/ .__/
                          /_/
Just Tap, Always Monitoring
WhaTap Python Agent version 0.1.48, 20171117

WHATAP_HOME: /Users/kimjihye/Downloads/django-react-blog-master/whatap
Config: /Users/kimjihye/Downloads/django-react-blog-master/whatap/whatap.conf
Logs: /Users/kimjihye/Downloads/django-react-blog-master/whatap/logs


[2017-11-23  04:59:19] : WA000 - WHATAP AGENT START!
[2017-11-23  04:59:19] :  - type object 'BaseHandler' has no attribute 'apply_response_fixes'
Traceback (most recent call last):
  File "/Users/kimjihye/Downloads/env/lib/python3.4/site-packages/whatap/trace/trace_import.py", line 29, in load_module
    IMPORT_HOOKS[fullname]['def'])(module)
  File "/Users/kimjihye/Downloads/env/lib/python3.4/site-packages/whatap/trace/mod/application_django.py", line 59, in instrument_handlers_base
    module.BaseHandler.apply_response_fixes)
AttributeError: type object 'BaseHandler' has no attribute 'apply_response_fixes'
[2017-11-23  14:34:31] :  - 
 _      ____       ______Python-AGENT  
| | /| / / /  ___ /_  __/__ ____
| |/ |/ / _ \/ _ `// / / _ `/ _ \
|__/|__/_//_/\_,_//_/  \_,_/ .__/
                          /_/
Just Tap, Always Monitoring
WhaTap Python Agent version 0.1.48, 20171117

WHATAP_HOME: /Users/kimjihye/workspace/django-blog/whatap
Config: /Users/kimjihye/workspace/django-blog/whatap/whatap.conf
Logs: /Users/kimjihye/workspace/django-blog/whatap/logs


[2017-11-23  14:34:31] : WA000 - WHATAP AGENT START!
[2017-11-23  05:34:32] :  - type object 'BaseHandler' has no attribute 'apply_response_fixes'
Traceback (most recent call last):
  File "/Users/kimjihye/workspace/django-blog/env/lib/python3.4/site-packages/whatap/trace/trace_import.py", line 29, in load_module
    IMPORT_HOOKS[fullname]['def'])(module)
  File "/Users/kimjihye/workspace/django-blog/env/lib/python3.4/site-packages/whatap/trace/mod/application_django.py", line 59, in instrument_handlers_base
    module.BaseHandler.apply_response_fixes)
AttributeError: type object 'BaseHandler' has no attribute 'apply_response_fixes'
[2017-11-23  05:34:32] :  - 
 _      ____       ______Python-AGENT  
| | /| / / /  ___ /_  __/__ ____
| |/ |/ / _ \/ _ `// / / _ `/ _ \
|__/|__/_//_/\_,_//_/  \_,_/ .__/
                          /_/
Just Tap, Always Monitoring
WhaTap Python Agent version 0.1.48, 20171117

WHATAP_HOME: /Users/kimjihye/workspace/django-blog/whatap
Config: /Users/kimjihye/workspace/django-blog/whatap/whatap.conf
Logs: /Users/kimjihye/workspace/django-blog/whatap/logs


[2017-11-23  05:34:32] : WA000 - WHATAP AGENT START!
[2017-11-23  05:34:32] :  - type object 'BaseHandler' has no attribute 'apply_response_fixes'
Traceback (most recent call last):
  File "/Users/kimjihye/workspace/django-blog/env/lib/python3.4/site-packages/whatap/trace/trace_import.py", line 29, in load_module
    IMPORT_HOOKS[fullname]['def'])(module)
  File "/Users/kimjihye/workspace/django-blog/env/lib/python3.4/site-packages/whatap/trace/mod/application_django.py", line 59, in instrument_handlers_base
    module.BaseHandler.apply_response_fixes)
AttributeError: type object 'BaseHandler' has no attribute 'apply_response_fixes'
[2017-11-23  05:34:36] : WA914 - [Errno 61] Connection refused
Traceback (most recent call last):
  File "/Users/kimjihye/workspace/django-blog/env/lib/python3.4/site-packages/whatap/trace/__init__.py", line 201, in get
    received = cls.s.recv(1024)
ConnectionRefusedError: [Errno 61] Connection refused
